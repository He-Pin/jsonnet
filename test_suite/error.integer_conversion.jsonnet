// Value just beyond MAX_SAFE_INTEGER (2^53)
local beyond_max = 9007199254740994;  // 2^53 + 1
beyond_max << 1  // Should throw error "numeric value outside safe integer range for bitwise operation"
