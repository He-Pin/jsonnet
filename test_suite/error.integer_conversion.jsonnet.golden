STATIC ERROR: error.integer_conversion.jsonnet:3:1-16: numeric value outside safe integer range for bitwise operation.
